src/Toolbar/LeftElement.react.js

```diff
@@ -90,12 +90,12 @@ class LeftElement extends PureComponent {
             easing: Easing.linear,
             useNativeDriver: Platform.OS === 'android',
         }).start(() => {
-            let leftElement = activate ? SEARCH_FORWARD_ICON : this.props.leftElement;
+            const leftElement = activate ? SEARCH_FORWARD_ICON : this.props.leftElement;
 
-            if (!this.state.leftElement) {
```

Comment:
are you sure this is the right fix?

Reply:
@habibridho Why it should fix this problem?